apps/web/vibes/soul/form/number-input/index.tsx

```diff
@@ -2,20 +2,22 @@
 
 import { clsx } from 'clsx';
 import { Minus, Plus } from 'lucide-react';
-import { forwardRef } from 'react';
+import * as React from 'react';
 
 import { FieldError } from '@/vibes/soul/form/field-error';
 import { Label } from '@/vibes/soul/form/label';
 
-export const NumberInput = forwardRef<
+export const NumberInput = React.forwardRef<
```

Comment:
How come the change to importing all of React instead of individual functions like we do elsewhere?

Reply:
All of the form inputs are doing a `* as React` import. I'll go through later and make it all consistent.